Source/WebCore/inspector/WorkerInspectorController.h

```diff
@@ -54,7 +55,7 @@ class WorkerInspectorController final : public Inspector::InspectorEnvironment {
 
     void workerTerminating();
 
-    void connectFrontend();
+    void connectFrontend(bool isAutomaticInspection = false, bool immediatelyPause = false, Function<void()>&& frontendInitializedCallback = { });
```

Comment:
We should avoid boolean parameters like these and use enumeration classes for clarity. Otherwise, the call sites can be confusing. e.g.:
```
enum class IsAutomaticInspection : bool { No, Yes };
```